profiler/options.go

```diff
@@ -252,6 +254,16 @@ func WithAgentlessUpload() Option {
 	}
 }
 
+// WithDeltaProfiles specifies if delta profiles are enabled. The default value
+// is true. This option takes precedence over the DD_PROFILING_DELTA
+// environment variable that can be set to "true" or "false" as well. See
+// https://dtdg.co/go-delta-profile-docs for more information.
+func WithDeltaProfiles(enabled bool) Option {
```

Comment:
You could consider `DisableDeltaProfiles()` without the bool arg. too. But not sure if better.

Reply:
Yeah, I considered that, and TBH I'm not sure if the bool is much better.

I went with the bool because:
- It makes it easier to dynamically configure this for clients (no need for an if statement if the logic can be expressed a boolean expression)
- The environment variable value can be overwritten. If we went with `DisableDeltaProfiles` and somebody disable delta profiles via an env var, the app would have no way to overwrite it in the code.

So I think I'll keep it as-is but I don't feel very strongly about it either way.

Reply:
I agree. Matching the env. var. sounds good to me. Resolving. Thanks